cpu/lpc2387/include/vendor/lpc23xx.h

```diff
@@ -424,6 +424,10 @@ Reset, and Code Security/Debugging */
 
 /* Reset, reset source identification */
 #define RSIR           (*(volatile unsigned long *)(SCB_BASE_ADDR + 0x180))
+#define RSIR_POR       (BIT0)
+#define RSIR_EXTR      (BIT1)
+#define RSIR_WDTR      (BIT2)
+#define RSIR_BODR      (BIT3)
```

Comment:
Is this really needed ? I don't like to modify vendor files.

Reply:
Me neither, but here the vendor file is really terse and has been modified / amended before. (Those `lpc23xx_XXX_t` structs are RIOT additions too)

Reply:
Accidentally hit the resolve button; and undid this now. I should stop looking at PR with my phone...

Reply:
Better remove them if not used by this PR.

Reply:
nvm this is partially used in `cpu_woke_from_backup()`
let's keep it this way then.